beacon-chain/sync/validate_aggregate_proof.go

```diff
@@ -41,6 +41,16 @@ func (s *Service) validateAggregateAndProof(ctx context.Context, pid peer.ID, ms
 		return pubsub.ValidationIgnore, nil
 	}
 
+	// We should not attempt to process this message if the node is running in optimistic mode.
+	// We just ignore in p2p so that the peer is not penalized.
+	status, err := s.cfg.chain.IsOptimistic(ctx)
```

Comment:
```suggestion
	optimistic, err := s.cfg.chain.IsOptimistic(ctx)
```

small nit, 
```go
if optimisitic {
```
seems more readable. Same applies to other gossip handlers